fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/derby/DerbyAdapter.java

```diff
@@ -228,6 +228,20 @@ public void alterSequenceRestartWith(String schemaName, String sequenceName, lon
         dropSequence(schemaName, sequenceName);
         createSequence(schemaName, sequenceName, restartWith, cache);
     }
+    
+    @Override
+    public void alterTableColumnIdentityCache(String schemaName, String tableName, String columnName, int cache) {
+        // Not supported by Derby
+        
+        final String qname = DataDefinitionUtil.getQualifiedName(schemaName, tableName);
+        DataDefinitionUtil.assertValidName(columnName);
+        
+        // modify the CACHE property of the identity column
+        final String ddl = "ALTER TABLE " + qname + " ALTER COLUMN " + columnName + " SET CACHE " + cache;
+
```

Comment:
```suggestion
```

If not supported, shouldn't this be removed?

Reply:
It's logged in the error so that we don't forget about it. We may need to consider something similar in the future for the other databases, but would need a completely different command.

Reply:
makes sense